nbs/core.ipynb

```diff
@@ -35,7 +35,7 @@
   {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #204.            dataframe_or_files_list: Optional[Union[DataFrame, SparkDataFrame, Sequence[str]]] = None,
</pre><p>Please restore this to <code>df</code>. Even though it's not a very good name anymore this is a breaking change. We can think of deprecating it in favor of another name in the future.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>